src/main/java/com/linkedin/kmf/services/ClusterTopicManipulationService.java

```diff
@@ -171,7 +175,28 @@ private boolean doesClusterContainTopic(String topic, Collection<Node> brokers,
 
       totalPartitionsInCluster += this.processBroker(logDirectoriesResponseMap, broker, topic);
     }
-    return totalPartitionsInCluster == expected;
+    return totalPartitionsInCluster == expected && ClusterTopicManipulationService.isTopicDescribeSuccessful(
```

Comment:
I should have pointed it out from the previous PR. Can you rename `expected` to `expectedTotalPartitionsInCluster` or something more descriptive?

Reply:
ack.